tests/integration/features/stable.feature

```diff
@@ -761,6 +761,27 @@ Feature: Testing the update scenario of stable releases
     M+LjFGQ3UrLdLEUspKcF0awiacjCuvMGF6ns835tSzzZA9EhlsTIcfpO0rLWZo2B
     h1nzVu4H9N0bSFDgo65oIQ==
     """
+  
+  Scenario: Updating the Nextcloud 21.0.0 to 22.0.0 on the stable channel
+    Given There is a release with channel "stable"
+    And The received version is "21.0.0.7"
+    And The received PHP version is "7.3.0"
+    And the installation mtime is "11"
+    When The request is sent
+    Then The response is non-empty
+    And Update to version "22.0.0.11" is available
+    And URL to download is "https://download.nextcloud.com/server/releases/nextcloud-22.0.0.zip"
```

Comment:
shouldn't it then also be `22.0.0.11` or change line `772`, no`?

Reply:
Maybe it should even be 22.1.0 ?
@blizzz ?

Reply:
> shouldn't it then also be 22.0.0.11 or change line 772, no`?

We don't use the 4th digit for the filenames :)